src/templates/dandelion/components/TokenSelector/TokenSelectorInstance.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import styled from 'styled-components'
+import { addressesEqual, GU } from '@aragon/ui'
+import { shortenAddress } from '../../../../web3-utils'
+import { ETHER_TOKEN_FAKE_ADDRESS } from '../../config/helpers/tokens'
+
+/* eslint-disable react/prop-types */
+const TokenSelectorInstance = React.memo(function TokenSelectorInstance({
+  address,
+  name,
+  symbol,
+  showIcon = true,
+}) {
+  return (
+    <div
+      css={`
+        display: flex;
+        align-items: center;
+      `}
+    >
+      {showIcon ? (
+        <Icon src={`https://chasing-coins.com/coin/logo/${symbol}`} />
+      ) : (
+        <div
+          css={`
+            width: ${3 * GU}px;
+          `}
+        />
+      )}
+      {symbol && (
+        <span
+          css={`
+            margin-right: ${1 * GU}px;
+          `}
+        >
+          {symbol}
+        </span>
+      )}
+      {name && (
+        <span
+          css={`
+            max-width: 110px;
+            margin-right: ${1 * GU}px;
+            overflow: hidden;
+            text-overflow: ellipsis;
+          `}
+        >
+          ({name})
+        </span>
+      )}
+      {!addressesEqual(address, ETHER_TOKEN_FAKE_ADDRESS) &&
+        shortenAddress(address)}
```

Comment:
We're missing closing braces

<img width="526" alt="Screen Shot 2020-02-12 at 10 42 04 PM" src="https://user-images.githubusercontent.com/4166642/74379934-f10af500-4de8-11ea-98e4-9786c87d919b.png">